denizkorkmaz@denizkorkmaz:~/PycharmProjects/CMPE492_Transfer_Learning_LSTM_v2_Test$ python3 main.py -finetuning
2020-12-03 21:42:27.591877: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory
2020-12-03 21:42:27.591897: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
Running: python3 /home/denizkorkmaz/PycharmProjects/CMPE492_Transfer_Learning_LSTM_v2_Test/model/CNN/train_cnn.py -finetuning
2020-12-03 21:42:29.075276: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory
2020-12-03 21:42:29.075303: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
rm: cannot remove '/home/denizkorkmaz/PycharmProjects/CMPE492_Transfer_Learning_LSTM_v2_Test/CNN_Model': No such file or directory
Found 1250 files belonging to 2 classes.
2020-12-03 21:42:31.289743: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcuda.so.1
2020-12-03 21:42:31.345455: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-12-03 21:42:31.345767: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:01:00.0 name: GeForce GTX 1050 computeCapability: 6.1
coreClock: 1.493GHz coreCount: 5 deviceMemorySize: 3.95GiB deviceMemoryBandwidth: 104.43GiB/s
2020-12-03 21:42:31.345958: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory
2020-12-03 21:42:31.346070: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcublas.so.10'; dlerror: libcublas.so.10: cannot open shared object file: No such file or directory
2020-12-03 21:42:31.346164: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcufft.so.10'; dlerror: libcufft.so.10: cannot open shared object file: No such file or directory
2020-12-03 21:42:31.346256: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcurand.so.10'; dlerror: libcurand.so.10: cannot open shared object file: No such file or directory
2020-12-03 21:42:31.346352: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcusolver.so.10'; dlerror: libcusolver.so.10: cannot open shared object file: No such file or directory
2020-12-03 21:42:31.346440: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcusparse.so.10'; dlerror: libcusparse.so.10: cannot open shared object file: No such file or directory
2020-12-03 21:42:31.346527: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudnn.so.7'; dlerror: libcudnn.so.7: cannot open shared object file: No such file or directory
2020-12-03 21:42:31.346552: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1753] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...
2020-12-03 21:42:31.360718: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2020-12-03 21:42:31.454405: I tensorflow/core/platform/profile_utils/cpu_utils.cc:104] CPU Frequency: 2199995000 Hz
2020-12-03 21:42:31.456402: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x60b21e0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-12-03 21:42:31.456465: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-12-03 21:42:31.477829: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-12-03 21:42:31.477895: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      
Found 350 files belonging to 2 classes.
Number of layers in the base model:  22
2020-12-03 21:42:33.305091: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 209715200 exceeds 10% of free system memory.
2020-12-03 21:42:33.463540: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 209715200 exceeds 10% of free system memory.
2020-12-03 21:42:33.515494: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 209715200 exceeds 10% of free system memory.
2020-12-03 21:42:34.062340: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [350]
	 [[{{node Placeholder/_3}}]]
2020-12-03 21:42:34.535168: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:118] None of the MLIR optimization passes are enabled (registered 1)
2020-12-03 21:42:37.504533: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 24883200 exceeds 10% of free system memory.
2020-12-03 21:42:37.715210: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 24883200 exceeds 10% of free system memory.
 6/20 [========>.....................] - ETA: 1:30 - loss: 0.5257 - accuracy: 0.6901WARNING:tensorflow:Your input ran out of data; interrupting training. Make sure that your dataset or generator can generate at least `steps_per_epoch * epochs` batches (in this case, 20 batches). You may need to use the repeat() function when building your dataset.
20/20 [==============================] - 39s 2s/step - loss: 0.5030 - accuracy: 0.7110
Initial Loss = 0.49, Initial Accuracy = 0.72
2020-12-03 21:43:12.627616: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [1250]
	 [[{{node Placeholder/_0}}]]
2020-12-03 21:43:12.649107: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [1250]
	 [[{{node Placeholder/_0}}]]
Epoch 1/15
20/20 [==============================] - ETA: 0s - loss: 0.4980 - accuracy: 0.73442020-12-03 21:46:01.285141: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [350]
	 [[{{node Placeholder/_0}}]]
2020-12-03 21:46:01.299859: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [350]
	 [[{{node Placeholder/_0}}]]
20/20 [==============================] - 205s 10s/step - loss: 0.4980 - accuracy: 0.7344 - val_loss: 2.5479 - val_accuracy: 0.4343
2020-12-03 21:46:38.091073: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [1250]
	 [[{{node Placeholder/_3}}]]
Epoch 2/15
20/20 [==============================] - ETA: 0s - loss: 0.2445 - accuracy: 0.89602020-12-03 21:49:47.578804: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [350]
	 [[{{node Placeholder/_3}}]]
2020-12-03 21:49:47.594447: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [350]
	 [[{{node Placeholder/_3}}]]
20/20 [==============================] - 223s 11s/step - loss: 0.2445 - accuracy: 0.8960 - val_loss: 3.2498 - val_accuracy: 0.4286
2020-12-03 21:50:21.009552: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [1250]
	 [[{{node Placeholder/_0}}]]
Epoch 3/15
20/20 [==============================] - ETA: 0s - loss: 0.1904 - accuracy: 0.91122020-12-03 21:53:11.100076: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [350]
	 [[{{node Placeholder/_0}}]]
2020-12-03 21:53:11.116381: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [350]
	 [[{{node Placeholder/_0}}]]
20/20 [==============================] - 206s 10s/step - loss: 0.1904 - accuracy: 0.9112 - val_loss: 3.4980 - val_accuracy: 0.4229
2020-12-03 21:53:46.782741: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [1250]
	 [[{{node Placeholder/_3}}]]
Epoch 4/15
20/20 [==============================] - ETA: 0s - loss: 0.1565 - accuracy: 0.92082020-12-03 21:56:45.237230: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [350]
	 [[{{node Placeholder/_3}}]]
2020-12-03 21:56:45.251887: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [350]
	 [[{{node Placeholder/_0}}]]
20/20 [==============================] - 211s 11s/step - loss: 0.1565 - accuracy: 0.9208 - val_loss: 3.4704 - val_accuracy: 0.4257
2020-12-03 21:57:18.025589: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [1250]
	 [[{{node Placeholder/_3}}]]
Epoch 5/15
20/20 [==============================] - ETA: 0s - loss: 0.1047 - accuracy: 0.96002020-12-03 22:00:03.513053: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [350]
	 [[{{node Placeholder/_0}}]]
2020-12-03 22:00:03.532150: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [350]
	 [[{{node Placeholder/_3}}]]
20/20 [==============================] - 198s 10s/step - loss: 0.1047 - accuracy: 0.9600 - val_loss: 3.5909 - val_accuracy: 0.4229
2020-12-03 22:00:35.785988: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [1250]
	 [[{{node Placeholder/_0}}]]
Epoch 6/15
20/20 [==============================] - ETA: 0s - loss: 0.1030 - accuracy: 0.95122020-12-03 22:03:21.633056: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [350]
	 [[{{node Placeholder/_3}}]]
2020-12-03 22:03:21.645580: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [350]
	 [[{{node Placeholder/_0}}]]
20/20 [==============================] - 197s 10s/step - loss: 0.1030 - accuracy: 0.9512 - val_loss: 3.8340 - val_accuracy: 0.4200
2020-12-03 22:03:52.614595: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [1250]
	 [[{{node Placeholder/_3}}]]
Epoch 7/15
20/20 [==============================] - ETA: 0s - loss: 0.0746 - accuracy: 0.97682020-12-03 22:06:28.513224: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [350]
	 [[{{node Placeholder/_3}}]]
2020-12-03 22:06:28.526011: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [350]
	 [[{{node Placeholder/_0}}]]
20/20 [==============================] - 186s 9s/step - loss: 0.0746 - accuracy: 0.9768 - val_loss: 3.9224 - val_accuracy: 0.4286
2020-12-03 22:06:59.084440: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [1250]
	 [[{{node Placeholder/_3}}]]
Epoch 8/15
20/20 [==============================] - ETA: 0s - loss: 0.0577 - accuracy: 0.98722020-12-03 22:09:56.994525: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [350]
	 [[{{node Placeholder/_3}}]]
2020-12-03 22:09:57.010428: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [350]
	 [[{{node Placeholder/_3}}]]
20/20 [==============================] - 213s 11s/step - loss: 0.0577 - accuracy: 0.9872 - val_loss: 4.1606 - val_accuracy: 0.4200
2020-12-03 22:10:31.637202: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [1250]
	 [[{{node Placeholder/_0}}]]
Epoch 9/15
20/20 [==============================] - ETA: 0s - loss: 0.0390 - accuracy: 0.99362020-12-03 22:13:21.583961: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [350]
	 [[{{node Placeholder/_3}}]]
2020-12-03 22:13:21.597640: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [350]
	 [[{{node Placeholder/_0}}]]
20/20 [==============================] - 203s 10s/step - loss: 0.0390 - accuracy: 0.9936 - val_loss: 4.4787 - val_accuracy: 0.4200
2020-12-03 22:13:54.262796: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [1250]
	 [[{{node Placeholder/_3}}]]
Epoch 10/15
20/20 [==============================] - ETA: 0s - loss: 0.0331 - accuracy: 0.99442020-12-03 22:16:45.049302: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [350]
	 [[{{node Placeholder/_0}}]]
2020-12-03 22:16:45.063361: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [350]
	 [[{{node Placeholder/_0}}]]
20/20 [==============================] - 204s 10s/step - loss: 0.0331 - accuracy: 0.9944 - val_loss: 4.5419 - val_accuracy: 0.4257
2020-12-03 22:17:18.046737: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [1250]
	 [[{{node Placeholder/_3}}]]
Epoch 11/15
20/20 [==============================] - ETA: 0s - loss: 0.0209 - accuracy: 1.00002020-12-03 22:20:15.277660: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [350]
	 [[{{node Placeholder/_0}}]]
2020-12-03 22:20:15.292208: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [350]
	 [[{{node Placeholder/_3}}]]
20/20 [==============================] - 212s 11s/step - loss: 0.0209 - accuracy: 1.0000 - val_loss: 4.6213 - val_accuracy: 0.4257
2020-12-03 22:20:49.598778: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [1250]
	 [[{{node Placeholder/_3}}]]
Epoch 12/15
20/20 [==============================] - ETA: 0s - loss: 0.0237 - accuracy: 0.99602020-12-03 22:23:45.591383: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [350]
	 [[{{node Placeholder/_0}}]]
2020-12-03 22:23:45.604630: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [350]
	 [[{{node Placeholder/_3}}]]
20/20 [==============================] - 212s 11s/step - loss: 0.0237 - accuracy: 0.9960 - val_loss: 4.8422 - val_accuracy: 0.4257
2020-12-03 22:24:22.074963: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [1250]
	 [[{{node Placeholder/_3}}]]
Epoch 13/15
20/20 [==============================] - ETA: 0s - loss: 0.0117 - accuracy: 1.00002020-12-03 22:27:17.917650: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [350]
	 [[{{node Placeholder/_3}}]]
2020-12-03 22:27:17.932204: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [350]
	 [[{{node Placeholder/_0}}]]
20/20 [==============================] - 212s 11s/step - loss: 0.0117 - accuracy: 1.0000 - val_loss: 4.9937 - val_accuracy: 0.4257
2020-12-03 22:27:53.867968: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [1250]
	 [[{{node Placeholder/_3}}]]
Epoch 14/15
20/20 [==============================] - ETA: 0s - loss: 0.0149 - accuracy: 0.99762020-12-03 22:30:48.337429: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [350]
	 [[{{node Placeholder/_0}}]]
2020-12-03 22:30:48.349987: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [350]
	 [[{{node Placeholder/_0}}]]
20/20 [==============================] - 206s 10s/step - loss: 0.0149 - accuracy: 0.9976 - val_loss: 5.0728 - val_accuracy: 0.4200
2020-12-03 22:31:20.017820: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [1250]
	 [[{{node Placeholder/_3}}]]
Epoch 15/15
20/20 [==============================] - ETA: 0s - loss: 0.0127 - accuracy: 0.99922020-12-03 22:34:07.429222: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [350]
	 [[{{node Placeholder/_3}}]]
2020-12-03 22:34:07.442891: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [350]
	 [[{{node Placeholder/_0}}]]
20/20 [==============================] - 200s 10s/step - loss: 0.0127 - accuracy: 0.9992 - val_loss: 5.3612 - val_accuracy: 0.4200
WARNING:tensorflow:From /home/denizkorkmaz/.local/lib/python3.6/site-packages/tensorflow/python/training/tracking/tracking.py:111: Model.state_updates (from tensorflow.python.keras.engine.training) is deprecated and will be removed in a future version.
Instructions for updating:
This property should not be used in TensorFlow 2.0, as updates are applied automatically.
2020-12-03 22:34:41.527113: W tensorflow/python/util/util.cc:348] Sets are not currently considered sequences, but this may change in the future, so consider avoiding using them.
WARNING:tensorflow:From /home/denizkorkmaz/.local/lib/python3.6/site-packages/tensorflow/python/training/tracking/tracking.py:111: Layer.updates (from tensorflow.python.keras.engine.base_layer) is deprecated and will be removed in a future version.
Instructions for updating:
This property should not be used in TensorFlow 2.0, as updates are applied automatically.
/usr/local/lib/python3.6/dist-packages/matplotlib/backends/backend_gtk3.py:195: Warning: Source ID 8 was not found when attempting to remove it
  GLib.source_remove(self._idle_draw_id)
Running: python3 /home/denizkorkmaz/PycharmProjects/CMPE492_Transfer_Learning_LSTM_v2_Test/model/LSTM/train_lstm.py
2020-12-03 22:34:44.541565: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory
2020-12-03 22:34:44.541588: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
rm: cannot remove '/home/denizkorkmaz/PycharmProjects/CMPE492_Transfer_Learning_LSTM_v2_Test/LSTM_Model': No such file or directory
rm: cannot remove '/home/denizkorkmaz/PycharmProjects/CMPE492_Transfer_Learning_LSTM_v2_Test/Final_Output.csv': No such file or directory
rm: cannot remove '/home/denizkorkmaz/PycharmProjects/CMPE492_Transfer_Learning_LSTM_v2_Test/features_training.csv': No such file or directory
rm: cannot remove '/home/denizkorkmaz/PycharmProjects/CMPE492_Transfer_Learning_LSTM_v2_Test/features_test.csv': No such file or directory
2020-12-03 22:34:46.213609: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcuda.so.1
2020-12-03 22:34:46.241384: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-12-03 22:34:46.241906: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:01:00.0 name: GeForce GTX 1050 computeCapability: 6.1
coreClock: 1.493GHz coreCount: 5 deviceMemorySize: 3.95GiB deviceMemoryBandwidth: 104.43GiB/s
2020-12-03 22:34:46.242019: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory
2020-12-03 22:34:46.242095: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcublas.so.10'; dlerror: libcublas.so.10: cannot open shared object file: No such file or directory
2020-12-03 22:34:46.242157: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcufft.so.10'; dlerror: libcufft.so.10: cannot open shared object file: No such file or directory
2020-12-03 22:34:46.242216: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcurand.so.10'; dlerror: libcurand.so.10: cannot open shared object file: No such file or directory
2020-12-03 22:34:46.242279: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcusolver.so.10'; dlerror: libcusolver.so.10: cannot open shared object file: No such file or directory
2020-12-03 22:34:46.242339: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcusparse.so.10'; dlerror: libcusparse.so.10: cannot open shared object file: No such file or directory
2020-12-03 22:34:46.242397: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudnn.so.7'; dlerror: libcudnn.so.7: cannot open shared object file: No such file or directory
2020-12-03 22:34:46.242408: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1753] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...
2020-12-03 22:34:46.242879: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2020-12-03 22:34:46.248297: I tensorflow/core/platform/profile_utils/cpu_utils.cc:104] CPU Frequency: 2199995000 Hz
2020-12-03 22:34:46.248861: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x47b0440 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-12-03 22:34:46.248875: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-12-03 22:34:46.253665: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-12-03 22:34:46.253684: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      
Found 1250 files belonging to 2 classes.
Found 350 files belonging to 2 classes.
2020-12-03 22:34:48.973099: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [1250]
	 [[{{node Placeholder/_0}}]]
2020-12-03 22:34:48.979237: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:118] None of the MLIR optimization passes are enabled (registered 1)
2020-12-03 22:34:48.988343: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [1250]
	 [[{{node Placeholder/_3}}]]
2020-12-03 22:34:54.381524: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 530841600 exceeds 10% of free system memory.
2020-12-03 22:34:54.493719: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 530841600 exceeds 10% of free system memory.
2020-12-03 22:35:01.074266: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 530841600 exceeds 10% of free system memory.
2020-12-03 22:35:01.205753: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 530841600 exceeds 10% of free system memory.
2020-12-03 22:35:07.877404: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 530841600 exceeds 10% of free system memory.
2020-12-03 22:36:53.640641: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [350]
	 [[{{node Placeholder/_0}}]]
2020-12-03 22:36:53.654109: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [350]
	 [[{{node Placeholder/_0}}]]
2020-12-03 22:37:35.204369: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:37:35.205207: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:37:35.206061: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
Epoch 1/20
2020-12-03 22:37:35.489025: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:37:35.490096: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:37:35.491119: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
2020-12-03 22:37:36.017783: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:37:36.018843: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:37:36.019839: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
1/1 [==============================] - ETA: 0s - loss: 0.7367 - accuracy: 0.60002020-12-03 22:37:38.884159: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:37:38.885153: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:37:38.886075: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
1/1 [==============================] - 4s 4s/step - loss: 0.7367 - accuracy: 0.6000 - val_loss: 0.6405 - val_accuracy: 0.7143
Epoch 2/20
1/1 [==============================] - 1s 1s/step - loss: 0.6895 - accuracy: 0.6000 - val_loss: 0.6351 - val_accuracy: 0.7143
Epoch 3/20
1/1 [==============================] - 1s 1s/step - loss: 0.6685 - accuracy: 0.6000 - val_loss: 0.6420 - val_accuracy: 0.7143
Epoch 4/20
1/1 [==============================] - 1s 1s/step - loss: 0.6638 - accuracy: 0.5600 - val_loss: 0.6484 - val_accuracy: 0.7143
Epoch 5/20
1/1 [==============================] - 1s 1s/step - loss: 0.6603 - accuracy: 0.6800 - val_loss: 0.6484 - val_accuracy: 0.7143
Epoch 6/20
1/1 [==============================] - 1s 1s/step - loss: 0.6507 - accuracy: 0.6800 - val_loss: 0.6429 - val_accuracy: 0.7143
Epoch 7/20
1/1 [==============================] - 1s 1s/step - loss: 0.6360 - accuracy: 0.6800 - val_loss: 0.6353 - val_accuracy: 0.7143
Epoch 8/20
1/1 [==============================] - 1s 1s/step - loss: 0.6200 - accuracy: 0.6800 - val_loss: 0.6284 - val_accuracy: 0.7143
Epoch 9/20
1/1 [==============================] - 1s 1s/step - loss: 0.6056 - accuracy: 0.7200 - val_loss: 0.6238 - val_accuracy: 0.7143
Epoch 10/20
1/1 [==============================] - 1s 1s/step - loss: 0.5941 - accuracy: 0.6400 - val_loss: 0.6213 - val_accuracy: 0.7143
Epoch 11/20
1/1 [==============================] - 1s 1s/step - loss: 0.5852 - accuracy: 0.6400 - val_loss: 0.6200 - val_accuracy: 0.7143
Epoch 12/20
1/1 [==============================] - 1s 1s/step - loss: 0.5773 - accuracy: 0.6400 - val_loss: 0.6188 - val_accuracy: 0.7143
Epoch 13/20
1/1 [==============================] - 1s 1s/step - loss: 0.5689 - accuracy: 0.6400 - val_loss: 0.6171 - val_accuracy: 0.7143
Epoch 14/20
1/1 [==============================] - 1s 1s/step - loss: 0.5591 - accuracy: 0.6400 - val_loss: 0.6150 - val_accuracy: 0.7143
Epoch 15/20
1/1 [==============================] - 1s 1s/step - loss: 0.5482 - accuracy: 0.6800 - val_loss: 0.6128 - val_accuracy: 0.7143
Epoch 16/20
1/1 [==============================] - 1s 1s/step - loss: 0.5370 - accuracy: 0.7600 - val_loss: 0.6111 - val_accuracy: 0.7143
Epoch 17/20
1/1 [==============================] - 1s 1s/step - loss: 0.5264 - accuracy: 0.7600 - val_loss: 0.6102 - val_accuracy: 0.7143
Epoch 18/20
1/1 [==============================] - 1s 998ms/step - loss: 0.5168 - accuracy: 0.8000 - val_loss: 0.6099 - val_accuracy: 0.7143
Epoch 19/20
1/1 [==============================] - 1s 1s/step - loss: 0.5083 - accuracy: 0.8000 - val_loss: 0.6097 - val_accuracy: 0.7143
Epoch 20/20
1/1 [==============================] - 1s 1s/step - loss: 0.5004 - accuracy: 0.8000 - val_loss: 0.6091 - val_accuracy: 0.7143
2020-12-03 22:37:59.219777: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:37:59.221150: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:37:59.222284: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
2020-12-03 22:37:59.401755: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:37:59.402751: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:37:59.403722: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
2020-12-03 22:37:59.583166: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:37:59.584177: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:37:59.585145: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
2020-12-03 22:37:59.749967: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:37:59.750962: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:37:59.751939: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
2020-12-03 22:37:59.915855: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:37:59.916817: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:37:59.917785: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
2020-12-03 22:38:00.086877: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:38:00.087933: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:38:00.088911: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
2020-12-03 22:38:00.267610: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:38:00.268618: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:38:00.269594: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
2020-12-03 22:38:00.445795: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:38:00.446760: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:38:00.447708: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
2020-12-03 22:38:00.613609: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:38:00.614585: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:38:00.615537: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
WARNING:tensorflow:From /home/denizkorkmaz/.local/lib/python3.6/site-packages/tensorflow/python/training/tracking/tracking.py:111: Model.state_updates (from tensorflow.python.keras.engine.training) is deprecated and will be removed in a future version.
Instructions for updating:
This property should not be used in TensorFlow 2.0, as updates are applied automatically.
2020-12-03 22:38:00.791341: W tensorflow/python/util/util.cc:348] Sets are not currently considered sequences, but this may change in the future, so consider avoiding using them.
WARNING:tensorflow:From /home/denizkorkmaz/.local/lib/python3.6/site-packages/tensorflow/python/training/tracking/tracking.py:111: Layer.updates (from tensorflow.python.keras.engine.base_layer) is deprecated and will be removed in a future version.
Instructions for updating:
This property should not be used in TensorFlow 2.0, as updates are applied automatically.
2020-12-03 22:38:00.994084: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:38:00.995046: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:38:00.996003: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
2020-12-03 22:38:01.235744: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:38:01.236719: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:38:01.237686: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
2020-12-03 22:38:01.433275: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:38:01.434271: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:38:01.435298: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
2020-12-03 22:38:01.595618: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:38:01.596585: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:38:01.597509: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
2020-12-03 22:38:01.771584: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:38:01.772586: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:38:01.773518: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
2020-12-03 22:38:01.934353: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:38:01.935333: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:38:01.936307: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
/usr/local/lib/python3.6/dist-packages/matplotlib/backends/backend_gtk3.py:195: Warning: Source ID 8 was not found when attempting to remove it
  GLib.source_remove(self._idle_draw_id)
Running: python3 /home/denizkorkmaz/PycharmProjects/CMPE492_Transfer_Learning_LSTM_v2_Test/model/CNN/test_cnn.py
2020-12-03 22:38:03.541560: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory
2020-12-03 22:38:03.541582: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
2020-12-03 22:38:05.054398: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcuda.so.1
2020-12-03 22:38:05.080780: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-12-03 22:38:05.081008: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:01:00.0 name: GeForce GTX 1050 computeCapability: 6.1
coreClock: 1.493GHz coreCount: 5 deviceMemorySize: 3.95GiB deviceMemoryBandwidth: 104.43GiB/s
2020-12-03 22:38:05.081085: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory
2020-12-03 22:38:05.081134: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcublas.so.10'; dlerror: libcublas.so.10: cannot open shared object file: No such file or directory
2020-12-03 22:38:05.081179: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcufft.so.10'; dlerror: libcufft.so.10: cannot open shared object file: No such file or directory
2020-12-03 22:38:05.081224: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcurand.so.10'; dlerror: libcurand.so.10: cannot open shared object file: No such file or directory
2020-12-03 22:38:05.081270: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcusolver.so.10'; dlerror: libcusolver.so.10: cannot open shared object file: No such file or directory
2020-12-03 22:38:05.081315: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcusparse.so.10'; dlerror: libcusparse.so.10: cannot open shared object file: No such file or directory
2020-12-03 22:38:05.081360: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudnn.so.7'; dlerror: libcudnn.so.7: cannot open shared object file: No such file or directory
2020-12-03 22:38:05.081367: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1753] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...
2020-12-03 22:38:05.081585: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2020-12-03 22:38:05.086740: I tensorflow/core/platform/profile_utils/cpu_utils.cc:104] CPU Frequency: 2199995000 Hz
2020-12-03 22:38:05.087108: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x574a080 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-12-03 22:38:05.087122: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-12-03 22:38:05.091327: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-12-03 22:38:05.091337: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      
Found 400 files belonging to 2 classes.
2020-12-03 22:38:07.459296: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [400]
	 [[{{node Placeholder/_3}}]]
2020-12-03 22:38:07.465429: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:118] None of the MLIR optimization passes are enabled (registered 1)
2020-12-03 22:38:07.493382: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'Placeholder/_3' with dtype int32 and shape [400]
	 [[{{node Placeholder/_3}}]]
Running: python3 /home/denizkorkmaz/PycharmProjects/CMPE492_Transfer_Learning_LSTM_v2_Test/model/LSTM/test_lstm.py
2020-12-03 22:38:49.317293: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory
2020-12-03 22:38:49.317316: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
2020-12-03 22:38:51.192712: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcuda.so.1
2020-12-03 22:38:51.220845: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-12-03 22:38:51.221096: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:01:00.0 name: GeForce GTX 1050 computeCapability: 6.1
coreClock: 1.493GHz coreCount: 5 deviceMemorySize: 3.95GiB deviceMemoryBandwidth: 104.43GiB/s
2020-12-03 22:38:51.221173: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory
2020-12-03 22:38:51.221221: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcublas.so.10'; dlerror: libcublas.so.10: cannot open shared object file: No such file or directory
2020-12-03 22:38:51.221266: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcufft.so.10'; dlerror: libcufft.so.10: cannot open shared object file: No such file or directory
2020-12-03 22:38:51.221310: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcurand.so.10'; dlerror: libcurand.so.10: cannot open shared object file: No such file or directory
2020-12-03 22:38:51.221354: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcusolver.so.10'; dlerror: libcusolver.so.10: cannot open shared object file: No such file or directory
2020-12-03 22:38:51.221399: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcusparse.so.10'; dlerror: libcusparse.so.10: cannot open shared object file: No such file or directory
2020-12-03 22:38:51.221443: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudnn.so.7'; dlerror: libcudnn.so.7: cannot open shared object file: No such file or directory
2020-12-03 22:38:51.221450: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1753] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...
2020-12-03 22:38:51.221653: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2020-12-03 22:38:51.226974: I tensorflow/core/platform/profile_utils/cpu_utils.cc:104] CPU Frequency: 2199995000 Hz
2020-12-03 22:38:51.227334: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x4309db0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-12-03 22:38:51.227351: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-12-03 22:38:51.231979: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-12-03 22:38:51.231990: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      
2020-12-03 22:38:51.248936: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_2_grad_concat_split_2_split_dim' with dtype int32
	 [[{{node gradients_split_2_grad_concat_split_2_split_dim}}]]
2020-12-03 22:38:51.249046: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_grad_concat_split_split_dim' with dtype int32
	 [[{{node gradients_split_grad_concat_split_split_dim}}]]
2020-12-03 22:38:51.249137: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_1_grad_concat_split_1_split_dim' with dtype int32
	 [[{{node gradients_split_1_grad_concat_split_1_split_dim}}]]
2020-12-03 22:38:51.321380: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_2_grad_concat_split_2_split_dim' with dtype int32
	 [[{{node gradients_split_2_grad_concat_split_2_split_dim}}]]
2020-12-03 22:38:51.321504: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_grad_concat_split_split_dim' with dtype int32
	 [[{{node gradients_split_grad_concat_split_split_dim}}]]
2020-12-03 22:38:51.321598: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_1_grad_concat_split_1_split_dim' with dtype int32
	 [[{{node gradients_split_1_grad_concat_split_1_split_dim}}]]
2020-12-03 22:38:51.364173: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_2_grad_concat_split_2_split_dim' with dtype int32
	 [[{{node gradients_split_2_grad_concat_split_2_split_dim}}]]
2020-12-03 22:38:51.364296: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_grad_concat_split_split_dim' with dtype int32
	 [[{{node gradients_split_grad_concat_split_split_dim}}]]
2020-12-03 22:38:51.364389: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_1_grad_concat_split_1_split_dim' with dtype int32
	 [[{{node gradients_split_1_grad_concat_split_1_split_dim}}]]
2020-12-03 22:38:51.442560: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_2_grad_concat_split_2_split_dim' with dtype int32
	 [[{{node gradients_split_2_grad_concat_split_2_split_dim}}]]
2020-12-03 22:38:51.442686: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_grad_concat_split_split_dim' with dtype int32
	 [[{{node gradients_split_grad_concat_split_split_dim}}]]
2020-12-03 22:38:51.442779: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_1_grad_concat_split_1_split_dim' with dtype int32
	 [[{{node gradients_split_1_grad_concat_split_1_split_dim}}]]
2020-12-03 22:38:51.746170: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_2_grad_concat_split_2_split_dim' with dtype int32
	 [[{{node gradients_split_2_grad_concat_split_2_split_dim}}]]
2020-12-03 22:38:51.746291: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_grad_concat_split_split_dim' with dtype int32
	 [[{{node gradients_split_grad_concat_split_split_dim}}]]
2020-12-03 22:38:51.746388: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_1_grad_concat_split_1_split_dim' with dtype int32
	 [[{{node gradients_split_1_grad_concat_split_1_split_dim}}]]
2020-12-03 22:38:51.774388: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_2_grad_concat_split_2_split_dim' with dtype int32
	 [[{{node gradients_split_2_grad_concat_split_2_split_dim}}]]
2020-12-03 22:38:51.774528: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_grad_concat_split_split_dim' with dtype int32
	 [[{{node gradients_split_grad_concat_split_split_dim}}]]
2020-12-03 22:38:51.774626: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_1_grad_concat_split_1_split_dim' with dtype int32
	 [[{{node gradients_split_1_grad_concat_split_1_split_dim}}]]
2020-12-03 22:38:51.848350: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_2_grad_concat_split_2_split_dim' with dtype int32
	 [[{{node gradients_split_2_grad_concat_split_2_split_dim}}]]
2020-12-03 22:38:51.848474: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_grad_concat_split_split_dim' with dtype int32
	 [[{{node gradients_split_grad_concat_split_split_dim}}]]
2020-12-03 22:38:51.848569: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_1_grad_concat_split_1_split_dim' with dtype int32
	 [[{{node gradients_split_1_grad_concat_split_1_split_dim}}]]
2020-12-03 22:38:51.891232: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_2_grad_concat_split_2_split_dim' with dtype int32
	 [[{{node gradients_split_2_grad_concat_split_2_split_dim}}]]
2020-12-03 22:38:51.891369: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_grad_concat_split_split_dim' with dtype int32
	 [[{{node gradients_split_grad_concat_split_split_dim}}]]
2020-12-03 22:38:51.891463: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_1_grad_concat_split_1_split_dim' with dtype int32
	 [[{{node gradients_split_1_grad_concat_split_1_split_dim}}]]
2020-12-03 22:38:52.112141: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_2_grad_concat_split_2_split_dim' with dtype int32
	 [[{{node gradients_split_2_grad_concat_split_2_split_dim}}]]
2020-12-03 22:38:52.112267: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_grad_concat_split_split_dim' with dtype int32
	 [[{{node gradients_split_grad_concat_split_split_dim}}]]
2020-12-03 22:38:52.112359: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_1_grad_concat_split_1_split_dim' with dtype int32
	 [[{{node gradients_split_1_grad_concat_split_1_split_dim}}]]
2020-12-03 22:38:52.185347: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_2_grad_concat_split_2_split_dim' with dtype int32
	 [[{{node gradients_split_2_grad_concat_split_2_split_dim}}]]
2020-12-03 22:38:52.185489: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_grad_concat_split_split_dim' with dtype int32
	 [[{{node gradients_split_grad_concat_split_split_dim}}]]
2020-12-03 22:38:52.185588: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_1_grad_concat_split_1_split_dim' with dtype int32
	 [[{{node gradients_split_1_grad_concat_split_1_split_dim}}]]
2020-12-03 22:38:52.283567: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_2_grad_concat_split_2_split_dim' with dtype int32
	 [[{{node gradients_split_2_grad_concat_split_2_split_dim}}]]
2020-12-03 22:38:52.283703: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_grad_concat_split_split_dim' with dtype int32
	 [[{{node gradients_split_grad_concat_split_split_dim}}]]
2020-12-03 22:38:52.283797: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients_split_1_grad_concat_split_1_split_dim' with dtype int32
	 [[{{node gradients_split_1_grad_concat_split_1_split_dim}}]]
2020-12-03 22:38:53.744745: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:38:53.746144: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:38:53.747473: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
2020-12-03 22:38:54.051288: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:118] None of the MLIR optimization passes are enabled (registered 1)
2020-12-03 22:38:54.255217: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_2_grad/concat/split_2/split_dim' with dtype int32
	 [[{{node gradients/split_2_grad/concat/split_2/split_dim}}]]
2020-12-03 22:38:54.256308: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_grad/concat/split/split_dim' with dtype int32
	 [[{{node gradients/split_grad/concat/split/split_dim}}]]
2020-12-03 22:38:54.257278: W tensorflow/core/common_runtime/executor.cc:1086] [/device:CPU:0] Executor start aborting: Invalid argument: You must feed a value for placeholder tensor 'gradients/split_1_grad/concat/split_1/split_dim' with dtype int32
	 [[{{node gradients/split_1_grad/concat/split_1/split_dim}}]]
Sigmoid Threshold: 0.5
Test Accuracy: 0.625
Recall Score: 0.6666666666666666
Precision Score: 0.5
F1 Score: 0.5714285714285715

/usr/local/lib/python3.6/dist-packages/matplotlib/backends/backend_gtk3.py:195: Warning: Source ID 8 was not found when attempting to remove it
  GLib.source_remove(self._idle_draw_id)
